controllers/chat/ChatController.js

```diff
@@ -525,9 +521,19 @@ module.exports = {
           };
         })
       );
+
+      const {betterChannelMember, betterChannelMemberObject} =
+        await BetterSocialCore.chat.updateBetterChannelMembers(channel, findOrCreateChannel, true, {
+          channel_type: CHANNEL_TYPE.ANONYMOUS
+        });
+
       await client.disconnectUser();
 
-      return res.status(200).json(findOrCreateChannel);
+      return res.status(200).json({
+        ...findOrCreateChannel,
+        better_channel_member: betterChannelMember,
+        better_channel_member_object: betterChannelMemberObject
+      });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here seem to introduce a new function call `updateBetterChannelMembers` and modify the response object returned by the API endpoint. Without more context, it's hard to say whether these changes are appropriate or not. However, from a code quality perspective, there don't appear to be any issues with this change.



<!-- This is an auto-generated comment by OSS CodeRabbit -->